openadapt/window/_windows.py

```diff
@@ -8,7 +8,7 @@
 import pickle
 
 
-def get_active_window_state() -> dict:
+def get_active_window_state(show_data) -> dict:
```

Comment:
How about:
```
def get_active_window_state(metadata_only=False) -> dict:
```

Reply:
As addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/290/commits/bb45d409eddf1eb26e21aaa00c27b3a02ccc5496, after deciding to implement the tray state machine, we no longer require any type of flag on the get_active_window_state function. Previously, the flag was utilized to expedite obtaining the active window state, as it was necessary for positioning the widget above the active window.